scripts/scil_compute_NODDI_priors.py

```diff
@@ -0,0 +1,190 @@
+                             add_overwrite_arg,
+                             add_verbose_arg)
+
+EPILOG = """
+Reference:
+    [1] Zhang H, Schneider T, Wheeler-Kingshott CA, Alexander DC.
+        NODDI: practical in vivo neurite orientation dispersion
+        and density imaging of the human brain.
+        NeuroImage. 2012 Jul 16;61:1000-16.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, epilog=EPILOG,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    p.add_argument('in_FA',
+                   help='Path to the FA volume.')
+    p.add_argument('in_AD',
+                   help='Path to the axial diffusivity (AD) volume.')
+    p.add_argument('in_MD',
+                   help='Path to the mean diffusivity (MD) volume.')
+
+    g1 = p.add_argument_group('Metrics options')
+    g1.add_argument(
+        '--fa_min', type=float, default='0.7',
+        help='Minimal threshold of FA (voxels above that threshold are '
+             'considered in the single fiber mask). [%(default)s]')
+    g1.add_argument(
+        '--fa_max', type=float, default='0.1',
+        help='Maximal threshold of FA (voxels under that threshold are '
+             'considered in the ventricles). [%(default)s]')
+    g1.add_argument(
+        '--md_min', dest='md_min',  type=float, default='0.003',
+        help='Minimal threshold of MD in mm2/s (voxels above that threshold '
+             'are considered for in the ventricles). [%(default)s]')
+
+    g2 = p.add_argument_group('Regions options')
+    g2.add_argument(
+        '--roi_radius', type=int, default=20,
+        help='Radius of the region used to estimate the priors. The roi will '
+             'be a cube spanning from ROI_CENTER in each direction. '
+             '[%(default)s]')
+    g2.add_argument(
+        '--roi_center', metavar='tuple(3)', nargs="+", type=int,
+        help='Center of the roi of size roi_radius used to estimate the '
+             'priors. [center of the 3D volume]')
+
+    g3 = p.add_argument_group('Outputs')
+    g3.add_argument(
+        '--output_1fiber', metavar='file',
+        help='Output path for the text file containing the single '
+             'fiber average value of AD. If not set, the file will not be '
+             'saved.')
+    g3.add_argument(
+        '--mask_output_1fiber', metavar='file',
+        help='Output path for single fiber mask. If not set, the mask will '
+             'not be saved.')
+    g3.add_argument(
+        '--output_ventricles', metavar='file',
+        help='Output path for the text file containing the ventricles average '
+             'value of MD. If not set, the file will not be saved.')
+    g3.add_argument(
+        '--mask_output_ventricles', metavar='file',
+        help='Output path for the ventricule mask. If not set, the mask will '
+             'not be saved.')
+
+    add_overwrite_arg(p)
+    add_verbose_arg(p)
+
+    return p
+
+
+def load(path):
```

Comment:
I don't like wrapper function for something like that or the save. I would just do it in the main like usual